src/test/javascript/spec/component/exam/manage/exam-management.component.spec.ts

```diff
@@ -17,6 +17,7 @@ import { Exam } from 'app/entities/exam.model';
 import { ExamManagementService } from 'app/exam/manage/exam-management.service';
 import { CourseManagementService } from 'app/course/manage/course-management.service';
 import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { ExamInformationDTO } from 'app/entities/exam-information.model';
 
```

Comment:
Could you do me a favor and also add this code to the file. This would also reset all mocks after every test and is a good practice :) 


```
   afterEach(function () {
        // completely restore all fakes created through the sandbox
        sinon.restore();
    });
```